keps/sig-storage/20180515-svcacct-token-volumes.md

```diff
@@ -159,6 +161,55 @@ sources:
     audience: ca.istio.io
 ```
 
+### File Permission
```

Comment:
Do we know how/if this interacts with SELinux?

@gnufied @jsafrane

Reply:
SELinux labels on files are orthogonal to UID/GIDs and are set by the container runtime when pods start.
https://github.com/kubernetes/enhancements/pull/1621 tries to change it, still, it's separate from file owners and permissions.